theano/tensor/tests/test_extra_ops.py

```diff
@@ -677,7 +677,23 @@ def setUp(self):
                     Unique(False, False, True),
                     Unique(True, False, True),
                     Unique(False, True, True),
-                    Unique(True, True, True)]
+                    Unique(True, True, True),
+                    Unique(axis=0),
+                    Unique(True, axis=0),
+                    Unique(False, True, axis=0),
+                    Unique(True, True, axis=0),
+                    Unique(False, False, True, axis=0),
+                    Unique(True, False, True, axis=0),
+                    Unique(False, True, True, axis=0),
+                    Unique(True, True, True, axis=0),
+                    Unique(axis=-1),
+                    Unique(True, axis=-1),
+                    Unique(False, True, axis=-1),
+                    Unique(True, True, axis=-1),
+                    Unique(False, False, True, axis=-1),
+                    Unique(True, False, True, axis=-1),
+                    Unique(False, True, True, axis=-1),
+                    Unique(True, True, True, axis=-1)]
```

Comment:
You might make the instances with axis into a second list that can be skipped when the required NumPy version is not present.

Reply:
You're right. I'll move these onto a different class that will be expected to fail if the numpy version is below 1.13